rollup.config.js

```diff
@@ -186,16 +186,24 @@ export default cliargs => [
     onwarn,
     watch
   },
-  // core
+  // core cjs, es
   {
     input: 'src/js/video.js',
-    output: {
-      format: 'cjs',
-      file: 'core.js',
-      strict: false,
-      banner,
-      globals: globals.module
-    },
+    output: [
+      {
+        format: 'es',
+        file: 'dist/alt/video.core.es.js',
+        strict: false,
+        banner,
+        globals: globals.module
+      }, {
+        format: 'cjs',
+        file: 'core.js',
```

Comment:
the reason we kept this top level is to make it easier for consumers to be able to do `require('video.js/core')`. We can probably expose the ES module file there as well, maybe? core.es.js? Then you could do `import videojs from 'video.js/core.es';`. Thoughts?

Reply:
@misteroneill thoughts?

Reply:
Yeah, I agree that having it at the same level as `core.js` makes sense.

Reply:
Good idea, I have just updated the output file name from `dist/alt/video.core.es.js` to `core.es.js`